### PR TITLE
Issue@1773

### PR DESCRIPTION
**Pull request by Gbahdeyboh**
Changed the rounded amounts on expenses email to their actual amounts as requested in #1773

### CODE REVIEW

File: templates/emails/collective.expense.approved.hbs [hunk 1]
```diff
@@ -30,7 +30,7 @@ Subject: Your expense to {{collective.name}} for {{{currency expense.amount curr
   </a>
 
   <br /><br />
-  <a href="{{actions.viewLatestExpenses}}" class="btn"><div>View Latest Expenses</div></a>
+  <a href="{{actions.viewLatestExpenses}}" class="btn"><div>View Expense</div></a>
```
Comment (Betree) [inline templates/emails/collective.expense.approved.hbs]: Why changing this label?

File: package-lock.json [hunk 2]
```diff
@@ -1,15274 +0,0 @@
-{
```
Comment (Betree) [inline package-lock.json]: Why deleting the `package-lock.json`?

**Comment by Betree** [comment]
Thanks for contributing @Gbahdeyboh! We usually don't commit the `package-lock.json` if the PR doesn't change any dependency.

Also can you change the title of the PR to have something more accurate?

Thanks!

**Comment by Gbahdeyboh** [comment]
Thanks for the feedback, i made a new PR that i think resolves the monetary amounts being rounded issue. 

https://github.com/opencollective/opencollective-api/pull/2392

Please kindly review.

**Comment by Betree** [comment]
Closing in favor of https://github.com/opencollective/opencollective-api/pull/2392

**Pull request closed without merging**